js/src/features/human-resouce/epics.ts

```diff
@@ -40,6 +48,108 @@ export const fetchAbsencesEpic: RootEpic = (action$, state$, { api }) =>
     ),
   );
 
+export const fetchMyAbsencesEpic: RootEpic = (action$, state$, { api }) =>
+  action$.pipe(
+    filter(isActionOf(fetchMyAbsencesAsync.request)),
+    switchMap(() =>
+      from(api.humanResource.fetchMyAbsences()).pipe(
+        map((result) =>
+          result.success
+            ? fetchMyAbsencesAsync.success(result.data)
+            : fetchMyAbsencesAsync.failure(result.reason),
+        ),
+      ),
+    ),
+  );
+
+export const refetchMyAbsencesEpic: RootEpic = (action$, state$, { api }) =>
+  action$.pipe(
+    filter(isActionOf(refetchMyAbsencesAsync.request)),
+    switchMap(() =>
+      from(api.humanResource.fetchMyAbsences()).pipe(
+        map((result) =>
+          result.success
+            ? refetchMyAbsencesAsync.success(result.data)
+            : refetchMyAbsencesAsync.failure(result.reason),
+        ),
+      ),
+    ),
+  );
+
+export const approveAbsenceEpic: RootEpic = (action$, state$, { api }) =>
+  action$.pipe(
+    filter(isActionOf(approveAbsenceAsync.request)),
+    switchMap((action) =>
+      from(api.humanResource.approveAbsence(action.payload)).pipe(
+        map((result) =>
+          result.success
+            ? approveAbsenceAsync.success()
+            : approveAbsenceAsync.failure(result.reason),
+        ),
+      ),
+    ),
+  );
+
+export const createAbsenceEpic: RootEpic = (action$, state$, { api }) =>
```

Comment:
Why after update you close form but after create not?

Reply:
create displays after list but update displays in modal.

Reply:
create displays after rows and update displays in modal. So modal we can close and inline form not